### PR TITLE
Document: showing links & targets of Intersphinx mapping file

### LINKED ISSUE #4244: Document python -msphinx as allowing to dump an intersphinx db
Subject: python -msphinx.ext.intersphinx is a practical way to dump an intersphinx db

### Problem
Sometimes a doc is missing an intersphinx db entry, causing headscratching for other projects that want to link to it.  `python -msphinx.ext.intersphinx url-or-path` is a practical, already existing way to print the contents of an intersphinx db, allowing one to quickly confirm that the entry is missing.  It would benefit from being documented at http://www.sphinx-doc.org/en/stable/ext/intersphinx.html.

#### Procedure to reproduce the problem
n/a

#### Error logs / results
n/a

#### Expected results
n/a

### Reproducible project / your project
n/a

### Environment info
n/a

### PR DESCRIPTION
**Pull request by TimKam**
Subject: Document: showing links & targets of Intersphinx mapping file

### Feature or Bugfix
- Docs

### Purpose
Closes #4244 

### Relates
#4244

### CODE REVIEW

File: doc/ext/intersphinx.rst [hunk 1]
```diff
@@ -84,7 +87,7 @@ linking:
    To add links to modules and objects in the Python standard library
    documentation, use::
 
-      intersphinx_mapping = {'python': ('https://docs.python.org/3.4', None)}
+      intersphinx_mapping = {'python': ('https://docs.python.org/3.6', None)}
```
Comment (tk0miya) [inline doc/ext/intersphinx.rst]: How about `https://docs.python.org/3` instead?

**Review by tk0miya — COMMENTED** [review]
+1!

File: doc/ext/intersphinx.rst (hunk 1 above)
Comment (TimKam) [reply doc/ext/intersphinx.rst]: Sure!

**Comment by tk0miya** [comment]
Thanks!

**Pull request merged**